iaslic-bulletin.csl

```diff
@@ -0,0 +1,397 @@
+              <text variable="genre" suffix=". "/>
+              <date variable="issued">
+                <date-part name="year" suffix=", "/>
+              </date>
+              <group delimiter="">
+                <label variable="page" form="short"/>
+                <text variable="page"/>
+              </group>
+              <group delimiter="">
+                <label variable="number-of-pages" form="short"/>
+                <text variable="number-of-pages"/>
+              </group>
+              <text value="unpublished" font-style="italic" prefix="(" suffix=")."/>
+            </group>
+          </else-if>
+          <else-if type="report">
+            <choose>
+              <if variable="genre">
+                <group delimiter=" ">
+                  <text variable="title" suffix="; " font-style="italic"/>
+                  <text variable="publisher-place" suffix="."/>
+                  <date variable="issued" prefix=" ">
+                    <date-part name="year"/>
+                    <date-part name="month" form="short" prefix=", "/>
+                    <date-part name="day" prefix=" " suffix="."/>
+                  </date>
+                  <text value="(press release)." font-style="italic"/>
+                </group>
+              </if>
+              <else-if variable="note">
+                <group delimiter=" ">
+                  <text variable="note" suffix=". "/>
+                  <text macro="title" font-style="italic" suffix=". "/>
+                  <text variable="number" prefix="Report No. " suffix=". "/>
+                  <date variable="issued">
+                    <date-part name="year" suffix="."/>
+                  </date>
+                  <text macro="publisher"/>
+                  <text macro="pages" prefix=" "/>
+                </group>
+              </else-if>
+              <else>
+                <group delimiter=" ">
+                  <text macro="title" font-style="italic" suffix=". "/>
+                  <text variable="number" suffix=". "/>
+                  <date variable="issued">
+                    <date-part name="year" suffix=". "/>
+                  </date>
+                  <text macro="publisher"/>
+                  <text macro="pages" prefix=" "/>
+                </group>
+              </else>
+            </choose>
+          </else-if>
+          <else-if type="webpage post post-weblog" match="any">
+            <group delimiter=" ">
+              <text macro="title" font-style="italic"/>
+              <text value="."/>
+              <date variable="issued" prefix=" (" suffix=").">
+                <date-part name="month" form="short"/>
+                <date-part name="day" prefix=" "/>
+                <date-part name="year" prefix=", "/>
+              </date>
+              <text variable="URL" suffix=". "/>
+              <group font-style="italic">
+                <text term="retrieved" prefix="(" suffix=" "/>
```

Comment:
This "retrieved+date" would be better put into a macro and then that macro can be easily called.